### ISSUE TITLE
UI WEB

### ISSUE THREAD

**Issue by vhdm** [expectocode/telegram-export#58]
@Lonami @expectocode 
can i create a web ui  for this dumper with django? it's possible?

**Comment by Lonami** [comment]
Yes, it is possible for you to create a web UI, if you really want to.

**Comment by Lonami** [comment]
Create a separate file for each user?

**Comment by vhdm** [comment]
@Lonami 
again error, :(
![cc](https://user-images.githubusercontent.com/14149777/41520266-0d83c042-72e2-11e8-9acb-f98cf51d4ae0.png)

**Comment by vhdm** [comment]
how to destroy current logged session?

**Comment by Lonami** [comment]
@vhdm Did you ever run --list-dialogs?

**Comment by vhdm** [comment]
@Lonami 
please help me again
thank you very much ❤️ 
![untitled](https://user-images.githubusercontent.com/14149777/41527169-881835ea-72fb-11e8-912f-10d551d54490.png)

**Comment by Lonami** [comment]
Well that's something I've never seen before, it would be helpful to know what message caused this issue.

**Comment by vhdm** [comment]
@Lonami 
Hi dear lonami
how to automatic fill confirm code in:
Please enter the code you received:
and two step verification password?

**Comment by Lonami** [comment]
That's no longer an issue with `telegram-export`, please ask it in the [right place](https://t.me/TelethonChat).

**Comment by expectocode** [comment]
@vhdm if you don't provide some logs or a way to recreate this bug, we can't fix it.

**Comment by Lonami** [comment]
Also, why is the issue title like that?

**Comment by expectocode** [comment]
@lonami because that's the errror in his new screenshot
